src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static final String VALID_USER_NAME = "validUser";
+    private static StorageDaoImpl storageDao;
+    private static RegistrationServiceImpl registrationService;
```

Comment:
You should use parent types here 
StorageDaoImpl -> StorageDao
RegistrationServiceImpl -> RegistrationService